playground/waitfor/WaitForSandbox.AppHost/Program.cs

```diff
@@ -17,7 +17,8 @@
 builder.AddProject<Projects.WaitForSandbox_ApiService>("api")
        .WithExternalHttpEndpoints()
        .WithReference(db).WaitFor(db)
-       .WaitForCompletion(dbsetup);
+       .WaitForCompletion(dbsetup)
+       .WithReplicas(2);
```

Comment:
You haven't tested waiting on projects that have replicas. I think that will wait on the first one to be ready.

Reply:
`WaitForCompletion` for a replica doesn't work: https://github.com/dotnet/aspire/blob/05cbb1cf5e446d5e6fd0c5ebac63fa708af3f964/src/Aspire.Hosting/ApplicationModel/ResourceNotificationService.cs#L188-L191

This is point 1 from the teams chat. Is it supposed to work? @mitchdenny

Reply:
It’s blocked explicitly but it’s possible to make it work in theory with this annotation

Reply:
Issue: https://github.com/dotnet/aspire/issues/6083